### PR TITLE
fix: clear table placeholders when data is empty

### PR DESCRIPTION
**Pull request by smakaiberriesandco**
https://github.com/ClosedXML/ClosedXML.Report/issues/338
https://github.com/ClosedXML/ClosedXML.Report/issues/343

### CODE REVIEW

**Comment by smakaiberriesandco** [comment]
when generating excel from a template with an empty list it throws an exception: `ClosedXML.Excel.Exceptions.EmptyTableException: Table 'Table1' should have at least 1 row.`

I updated the commit to fix that issue too.

File: ClosedXML.Report/ClosedXML.Report.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net8.0</TargetFrameworks>
```
Comment (Pankraty) [inline ClosedXML.Report/ClosedXML.Report.csproj]: Why do we need to add another target framework if it works fine with `nestandard2.1`?

File: tests/ClosedXML.Report.Tests/RangeInterpreterTests.cs [hunk 2]
```diff
@@ -108,7 +109,7 @@ public void ShouldNotThrowExceptionIfAccessSomeChildrenNullProp()
         public void ShouldDestroyEmptyTable()
```
Comment (Pankraty) [inline tests/ClosedXML.Report.Tests/RangeInterpreterTests.cs:145]: The test is broken now

**Review by Pankraty — CHANGES_REQUESTED** [review]

**Comment by smakaiberriesandco** [comment]
Apologies for any confusion caused earlier (The exception mentioned above occurs only when the table range is one row larger than the row containing the placeholders). 

So when table range is correct and we provide an empty list to the table, the results still display the placeholders:
image
With my modifications, the table rows are now cleared:
image
Test created, All tests have been successfully executed.

File: tests/ClosedXML.Report.Tests/RangeInterpreterTests.cs (hunk 2 above)
Comment (smakaiberriesandco) [reply tests/ClosedXML.Report.Tests/RangeInterpreterTests.cs:145]: fixed

**Comment by Pankraty** [comment]
Thanks for the contribution and sorry it took me so long!

**Pull request merged**